cachecontrol/serialize.py

```diff
@@ -25,10 +25,16 @@ def _b64_decode_str(s):
 
 
 class Serializer(object):
-
-    def dumps(self, request, response, body):
+    def dumps(self, request, response, body=None):
```

Comment:
Don't ned to revert this line, just explicitly pass `None` from `update_cached_response`?